wp/lib/bap_wp/src/precondition.mli

```diff
@@ -278,6 +278,7 @@ val visit_sub : Env.t -> Constr.t -> Bap.Std.Sub.t -> Constr.t * Env.t
 val check
   : ?refute:bool
   -> ?print_constr: (string list)
+  -> ?debug: (string list)
```

Comment:
Does `debug` take the same options as `eval`? While we're here, can we also add the `print_constr` options to the doc as well?

Reply:
I knew you would catch that one next! Already done!

Reply:
> I'm personally fine with them being printed. But that's because i can never remember where logs go.

I'm fine with either as well.

Reply:
A question on comments. Since the debug-flags, by design, break the black-box nature of functions they are called in, can the comments concerning the debug flags also be allowed to talk about the function's inner implementation, or should they try to keep things opaque to respect the interface.

Reply:
I would say go ahead, but there is no need to get too detailed as long as you're able to mention the options and what each one prints out.